src/lib/realmdb/RealmDB.js

```diff
@@ -1,29 +1,32 @@
 //@flow
-import { Collection, Database } from '@textile/threaddb'
-import * as TextileCrypto from '@textile/crypto'
-import { once, sortBy } from 'lodash'
+import { assign, once, sortBy } from 'lodash'
 import * as Realm from 'realm-web'
-import Config from '../../config/config'
+import TextileCrypto from '@textile/crypto'
+
 import { JWT } from '../constants/localStorage'
+import AsyncStorage from '../utils/asyncStorage'
+
+import Config from '../../config/config'
 import logger from '../logger/pino-logger'
-import { FeedItemSchema } from '../textile/feedSchema' // Some json-schema.org schema
+
+import type { ThreadDB } from '../textile/ThreadDB'
 import type { ProfileDB } from '../userStorage/UserProfileStorage'
 import type { DB } from '../userStorage/UserStorage'
 import type { Profile } from '../userStorage/UserStorageClass'
-import AsyncStorage from '../utils/asyncStorage'
 import type { TransactionDetails } from '../userStorage/FeedStorage'
 
 const log = logger.child({ from: 'RealmDB' })
+
 class RealmDB implements DB, ProfileDB {
```

Comment:
revert all non functional syntax changes.
you can keep the init(frontenddb) and types but that's it. absolutly nothing else.
this.Feed.table syntax should be restored

Reply:
its ok. u just want the old syntax back. no Feed and FeedTable
table should be accessed via Feed.table
and i also dont want any other syntax only refactoring

Reply:
aaah ok understood. will update it